scripts/test-unit

```diff
@@ -21,6 +21,7 @@ pushd "${GOPATH}" > /dev/null
 
   pushd "src/bpm" > /dev/null
     go generate ./...
+    go install ./cmd/bpm
```

Comment:
Why is this needed? Don't the tests install bpm using `gexec`?

Reply:
I think I added it to get faster feedback when running all of the tests if there was a compilation error. I'll remove it as it forces the entire codebase to compile even if you're working on a single package.